Makefile

```diff
@@ -68,7 +68,7 @@ endif
 
 	mv $(COORDINATOR) $(PLATFORM_DIST_PATH)/bin
 	mv $(LOADTEST) $(PLATFORM_DIST_PATH)/bin
-	tar -czf $(DIST_PATH)/mattermost-load-test-ng_$(DIST_VER)_$(PLATFORM).tar.gz $(PLATFORM_DIST_PATH)
+	tar -C $(PLATFORM_DIST_PATH) -czf $(DIST_PATH)/mattermost-load-test-ng_$(DIST_VER)_$(PLATFORM).tar.gz ./
```

Comment:
Should we also provide a `.zip` version?

Reply:
It is generally provided for MS-Windows releases. I don't remember if I ever downloaded a `zip` for a linux release so far 😅

Reply:
Fair enough. Let's keep the geeky vibe alive :)